tests/unit/services/scheduler-test.js

```diff
@@ -0,0 +1,63 @@
+import Ember from 'ember';
```

Comment:
Since we're adding tests here, can we add a test for `cancelTask` to keep the tests balanced? Also, do we want to cover other methods in the service (yes, yes we do :))

Reply:
When you mentioned `cancelTask`, are you speaking of the `cancelWork` method?

Reply:
Yes :)

Too many addons...